test/VoteManager.js

```diff
@@ -389,6 +389,7 @@ describe('VoteManager', function () {
       it('should not be able to commit if stake is below minstake', async function () {
         await mineToNextEpoch();
         const epoch = await getEpoch();
+        await parameters.setMinStake(tokenAmount('100'));
```

Comment:
Why change the parameters? Change the staked amount instead.
Adhoc changes to the parameter need to be avoided. This test is not about parameters contract.

Reply:
I had suggested changing the min stake as I assumed otherwise. This is fair, we should change the staked amount itself. cc: @GauravJain9

Reply:
yeah